src/textual/dom.py

```diff
@@ -13,6 +13,7 @@
     TypeVar,
     cast,
     overload,
+    Union,
```

Comment:
Is this used?

Reply:
Good catch, nope. Did a lot of changes in there but backed them out in the end so that'll be a hangover.